.github/workflows/java-gradle-library.yaml

```diff
@@ -59,44 +59,31 @@ on:
         required: true
 
 jobs:
-  build:
-    name: Build
+  java-gradle-base:
```

Comment:
I would name it build. Or build-and-test

Reply:
@MichaelKora What about the renaming?